File: URI/1012.c
#include <stdio.h>

float main (){
  
  double a, b, c, triangulo, circulo, trapezio, quadrado, retangulo, pi;
  pi = 3.14; //Assumindo pi com esse valor 
  
  scanf("%lf %lf %lf", &a, &b, &c);

  triangulo = (a*c)/2;
  circulo = pi*c*c;
  trapezio = ((a + b)*c)/2;
  quadrado = b*b;
  retangulo = a*b;

  printf("TRIANGULO: %.3f\n", triangulo);
  printf("CIRCULO: %.3f\n", circulo);
  printf("TRAPEZIO: %.3f\n", trapezio);
  printf("QUADRADO: %.3f\n", quadrado);
  printf("RETANGULO: %.3f\n", retangulo);

}

File: URI/1012.py
#1012
valores = input().split(" ")

# a = float(valores[0])
# b = float(valores[1])
# c = float(valores[2])

triangulo = float(valores[0]) * float(valores[2]) / 2
circulo = 3.14159 * (float(valores[2])) ** 2
trapezio = (float(valores[0]) + float(valores[1])) * float(valores[2]) / 2
quadrado = float(valores[1]) ** 2
retangulo = float(valores[0]) * float(valores[1])

print(f"TRIANGULO: {triangulo:.3f}")
print(f"CIRCULO: {circulo:.3f}")
print(f"TRAPEZIO: {trapezio:.3f}")
print(f"QUADRADO: {quadrado:.3f}")
print(f"RETANGULO: {retangulo:.3f}")
